packages/hooks/src/useCallbackRef/useCallbackRef.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useMemo, useRef } from "react";
+
+/**
+ * A custom hook that converts a callback to a ref to avoid triggering re-renders when passed as a
+ * prop or avoid re-executing effects when passed as a dependency
+ */
+export function useCallbackRef<T extends (...args: unknown[]) => unknown>(
```

Comment:
I see that this only used in `useOnMount` but I am curious if you think it's worth surfacing this as a story? I'm not sure so don't take my question as something I'm alluding to 😆  Just curious if you think there's a scenario where someone looks at Hooks in storybook and thinks, _there's no useCallbackRef_ and then creates it again for another purpose? Or do you think this would really only be used in `useOnMount` and it's not worth surfacing?

Reply:
This is a good suggestion. I didn't originally because there wouldn't really be any functionality to show in Storybook so it didn't seem like documenting it would really help folks learn how to use the hook and I left that out. But looking at it from a discoverability perspective it would definitely be beneficial for folks to know it is there. So I added some simple documentation 😄

Reply:
Awesome - thank you for adding that!